test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/otel-collector-config.yaml

```diff
@@ -16,13 +16,39 @@ receivers:
       grpc:
         endpoint: 0.0.0.0:5317
         tls:
-          cert_file: /cfg/otel-collector.crt
-          key_file: /cfg/otel-collector.key
+          cert_file: /obj/Debug/otel-test-server-cert.pem
```

Comment:
Does the test only ever run in Debug mode?

Reply:
Resolved, tyvm.